docs/api.rst

```diff
@@ -5763,6 +5769,14 @@ InteractionReference
 .. autoclass:: InteractionReference
     :members:
 
+RoleSubscriptionData
```

Comment:
Isn't this a Discord Model? This is currently under data classes

Reply:
Yup, I mistakenly just put it right next to `InteractionReference`, which turned out to also be in the wrong section.
moved them in https://github.com/DisnakeDev/disnake/pull/913/commits/b150cd14973934926f8f5da7f67c5386a7e33351